ale_linters/perl/perlcritic.vim

```diff
@@ -1,17 +1,32 @@
-" Author: Vincent Lequertier <https://github.com/SkySymbol>
+" Author: Vincent Lequertier <https://github.com/SkySymbol>, Chris Weyl <cweyl@alumni.drew.edu>
 " Description: This file adds support for checking perl with perl critic
 
-if !exists('g:ale_perl_perlcritic_showrules')
-    let g:ale_perl_perlcritic_showrules = 0
-endif
+let g:ale_perl_perlcritic_executable =
+\   get(g:, 'ale_perl_perlcritic_executable', 'perlcritic')
+
+let g:ale_perl_perlcritic_profile =
+\   get(g:, 'ale_perl_perlcritic_profile', '.../.perlcriticrc')
```

Comment:
Thanks for that -- I'd mistaken the `...` as something `perlcritic` does, rather than a shell construct.